tools/scenarios-plugin.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const scenariosPlugin = () => {
+  return {
+    name: 'scenarios',
+    load(id) {
+      if (id.includes('bson_corpus_test_loader.js')) {
+        const corpus = fs
```

Comment:
Not super necessary, but would it be possible to `require` in `test/tools/bson_corpus_test_loader` here?

Reply:
We did consider commonly using the test loader script. However, the loader script is calling `JSON.parse` in the map stage to output an object, whereas the rollup plugin expects a string output. In the rollup plugin, we would have then had to essentially undo the `JSON.parse`, i.e. `JSON.stringify(JSON.parse(fs.readFileSync(...))` which seemed unnecessarily repetitive. Alternatively, we could have had the loader script not use `JSON.parse`, and then have it only be parsed into an object in the non-browser case, but that would have involved logic in the main test file beyond `require('./tools/bson_corpus_test_loader');`, which also seemed undesirable.

Reply:
I don't think the extra `JSON.stringify` is that big of a deal. This is a build-time step, so it being super-efficient is not really important.

Reply:
Should we make that update in js-bson too? It's already been merged, but I can open another pr.

Reply:
@shsaskin yeah, you can add that PR too.